src/agentscope/web/workstation/workflow_node.py

```diff
@@ -1853,6 +1857,189 @@ def run(self, *args, **kwargs):
+
+        if 'model' not in params_dict['settings']:
+            raise Exception("model not found in settings")
+        if 'headers' not in params_dict['settings']:
+            raise Exception("headers key not found in settings")
+
+        # 调用内部函数，将LLM Model请求参数组合为大模型可以识别的参数
+        params_dict = self.convert_llm_request(params_dict)
+        self.api_url = params_dict['settings']['url']
+        self.api_header = params_dict['settings']['headers']
+        if not isinstance(self.api_header, dict):
+            raise Exception(f"header:{self.api_header} type is not dict")
+
+        for i, param_spec in enumerate(params_dict['inputs']):
+            param_spec.setdefault('extra', {})
+            param_spec['extra'].setdefault('location', '')
+            if param_spec['extra'].get('location', '') not in {'query', 'body'}:
+                raise Exception("input param: {param_spec} extra-location not found('query' or 'body')")
+
+        for i, param_spec in enumerate(params_dict['outputs']):
+            param_one_dict = ASDiGraph.generate_node_param_spec(param_spec)
+            self.output_params_spec = param_one_dict
+
+        return {
+            "imports": "",
+            "inits": "",
+            "execs": "",
+        }
+
+    def __call__(self, *args, **kwargs):
+        # 注意，入参为空，表示上一个节点没有运行成功，这里简单起见，当前节点认为尚未运行
+        if len(kwargs) == 0:
+            self.running_status = 'init'
+            return {}
+
+        self.running_status = 'running'
+        try:
+            self.run(*args, **kwargs)
+            self.running_status = 'success'
+            return self.output_params
+        except Exception as err:
+            logger.error(f'{traceback.format_exc()}')
+            self.running_status = 'failed'
+            self.running_message = f'{repr(err)}'
+            return {}
+
+    def run(self, *args, **kwargs):
+        params_pool = self.dag_obj.params_pool
+        params_pool.setdefault(self.node_id, {})
+
+        # 1. 建立参数映射
+        params_dict = self.opt_kwargs
+
+        for i, param_spec in enumerate(params_dict['inputs']):
+            param_one_dict_for_query, param_one_dict_for_body \
+                = self.dag_obj.generate_node_param_real_for_api_input(param_spec)
+
+            if not isinstance(param_one_dict_for_query, dict):
+                raise Exception("input param: {param_one_dict_for_query} type not dict")
+            if not isinstance(param_one_dict_for_body, dict):
+                raise Exception("input param: {param_one_dict_for_body} type not dict")
+
+            self.input_params |= param_one_dict_for_query
+            self.input_params |= param_one_dict_for_body
+            self.input_params_for_query |= param_one_dict_for_query
+            self.input_params_for_body |= param_one_dict_for_body
+
+        # 大模型参数校验
+        if 'temperature' in self.input_params_for_body and not (0 <= self.input_params_for_body['temperature'] <= 1):
```

Comment:
temperature = self.input_params_for_body.get('self.input_params_for_body', None)
if not temperature or (0 <= temperature  <= 1):
    raise e